### ISSUE TITLE
Astro + Cloudflare = JAM Comments Incompatibility

### ISSUE THREAD

**Issue by cameronapak** [alexmacarthur/jam-comments-javascript#9]
Hey, love what you're doing with JAM Comments. Super cool project. Adding it to faith.tools. 

This issue is non-urgent, but I'm intrigued if you have a quick answer. 

I use Cloudflare. Things work perfect locally. When I build, I get an error that says:

```bash
✘ [ERROR] Could not resolve "fs"

    node_modules/@jam-comments/server-utilities/dist/esm/utils.js:1:74:
      1 │ import { existsSync, mkdirSync, readFileSync, rmdirSync, writeFile } from "fs";
        ╵                                                                           ~~~~

  The package "fs" wasn't found on the file system but is built into node. Are you trying to bundle
  for node? You can use "platform: 'node'" to do that, which will remove this error.

Could not resolve "fs"
```

Makes sense that it's happening. I didn't know if you had any alternative option to get it working. I may switch back to Netlify _(as I had no real reason to use Cloudflare)_ to get this working. Jam comments is awesome!

Some relevant packages are `"astro": "^4.11.0"` and `"@astrojs/cloudflare": "9.2.1"`.

**Comment by cameronapak** [comment]
I decided to switch back to Netlify from Cloudflare and it works like a charm :)

**Comment by alexmacarthur** [comment]
Fixed in v2.8.1! It'll now work in both dynamically and statically rendered setups. 👍

**Comment by cameronapak** [comment]
Thank you, man! 

And hey, I temporarily removed JAM Comments from faith.tools because how I was using it and how people were using it was different. I wanted people to share stories of apps they're using and then most people wrote comments to ask questions about the apps, haha. Until I can find a better way to implement that, I will have JAM Comments disabled on faith.tools. (It's still in my back pocket when needed. Very thankful for you and what you've created!)
